modules/database/src/ioc/db/Makefile

```diff
@@ -11,10 +11,12 @@
 
 SRC_DIRS += $(IOCDIR)/db
 
+INC += arrayRangeModifier.h
```

Comment:
Does `arrayRangeModifier.h` need to be public API?

Reply:
`wrapArrayIndices` is used in the array filter code, otherwise: no. I could give this function its own C and header file. Or perhaps you have a better idea?

Reply:
If this header doesn't need to be public, then simply remove the `INC += `.